modules/renter/hostdb/dependencies.go

```diff
@@ -12,63 +11,34 @@ import (
 // These interfaces define the HostDB's dependencies. Using the smallest
 // interface possible makes it easier to mock these dependencies in testing.
 type (
-	consensusSet interface {
-		ConsensusSetSubscribe(modules.ConsensusSetSubscriber, modules.ConsensusChangeID) error
-	}
-
-	dialer interface {
-		DialTimeout(modules.NetAddress, time.Duration) (net.Conn, error)
-	}
-
-	sleeper interface {
-		Sleep(time.Duration)
-	}
-
-	persister interface {
-		save(hdbPersist) error
-		saveSync(hdbPersist) error
-		load(*hdbPersist) error
+	dependencies interface {
+		dialTimeout(modules.NetAddress, time.Duration) (net.Conn, error)
+		disrupt(string) bool
+		loadFile(persist.Metadata, interface{}, string) error
+		saveFile(persist.Metadata, interface{}, string) error
+		saveFileSync(persist.Metadata, interface{}, string) error
+		sleep(time.Duration)
```

Comment:
I still feel bad about this. The persister object was actually a pretty good idea, made some things a lot nicer. I got rid of it because it was unfamiliar, but in the future would be open to bringing things like that back. Though it did strike me as too much on the side of clever.